src/components/MoviesSearchContainer/MoviesSearchContainer.js

```diff
@@ -16,9 +16,9 @@ const MoviesSearchContainer = ({ searchQuery }) => {
 
   return (
     <div className="movies-search-container">
```

Comment:
Since there's no `MovieSearch` component, you can rename it. This should make the naming convention simpler for you.